modules/dnn/test/test_model.cpp

```diff
@@ -370,6 +389,21 @@ TEST_P(Test_Model, Segmentation)
     testSegmentationModel(weights_file, config_file, inp, exp, norm, size, mean, scale, swapRB);
 }
 
+TEST_P(Test_Model, SceneTextRec)
+{
+    std::string imgPath = _tf("welcome.png");
```

Comment:
Where is this file?

Reply:
I have already put the data into opencv/opencv_extra, but it has not been merged yet.
https://github.com/opencv/opencv_extra/pull/773
The name of image has been changed into "text_rec_test.png", and I will push again when the data is ready.
Thanks for your review.

Reply:
Please push the new name. You don't need to wait the merge. See [this step](https://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/26892/steps/Merge%20opencv_extra%20with%20test%20branch/logs/stdio) in the build pipeline.

Reply:
Thank you.
Sadly, I still get an error in "Linux x64 Debug", but I find that the test_dnn in "Linux x64" passed. I am not sure about the difference between these two tests.

Is there any detailed log information containing which line throws out the error.
I can only get "error: (-215: Assertion failed) dims <= 2 in function 'at' thrown in the test body." in https://pullrequest.opencv.org/buildbot/builders/precommit_linux64_no_opt/builds/24676/steps/test_dnn/logs/stdio

I think I have tested the API successfully, and you can see more information in https://github.com/HannibalAPE/opencv/blob/text_det_recog_demo/doc/tutorials/dnn/dnn_scene_text_det_and_rec/scene_text_recognition.markdown

Reply:
> I still get an error in "Linux x64 Debug", but I find that the test_dnn in "Linux x64" passed. I am not sure about the difference between these two tests.

Obviously, it is "Debug" mode (with extra checks).

Reply:
@alalek Thanks for your reminder.